includes/templates/admin/edit-story.php

```diff
@@ -24,22 +24,78 @@
  * limitations under the License.
  */
 
+use Google\Web_Stories\Page_Template_Post_Type;
+
 // don't load directly.
 if ( ! defined( 'ABSPATH' ) ) {
 	die( '-1' );
 }
 
 global $post_type, $post_type_object, $post;
 
-$rest_base = ! empty( $post_type_object->rest_base ) ? $post_type_object->rest_base : $post_type_object->name;
-$demo      = ( isset( $_GET['web-stories-demo'] ) && (bool) $_GET['web-stories-demo'] ) ? 'true' : 'false'; // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+$stories_rest_base = ! empty( $post_type_object->rest_base ) ? $post_type_object->rest_base : $post_type_object->name;
```

Comment:
You think we should move this logic some to the editor class, similar to how it works with the dashboard class?

Reply:
Hmm maybe. Not sure how useful it is. We can look at this in a new ticket.